jenkins-slaves/jenkins-slave-golang/Dockerfile

```
@@ -13,6 +13,8 @@ RUN mkdir -p /usr/src/go/src/redhat && \
     chown -R 1001 /usr/local/go && \
     rm -f /usr/go${GO_VERSION}.linux-amd64.tar.gz
 
+RUN go get -u github.com/golang/dep/cmd/dep
```

Can you move this into the above `RUN` statement, before the chown commands? Your `RUN` statement will create `/usr/src/go/src/github.com` as `root` thus preventing any other dependencies to be pulled in.
I also suggest to change permissions on `${HOME}/.cache/go-build` directory to avoid warnings and disabling the cache (see below)
```suggestion
RUN mkdir -p /usr/src/go/src/redhat && \
    tar -xzf /usr/go${GO_VERSION}.linux-amd64.tar.gz && \
    mv $(pwd)/go /usr/local/ && \
    go get -u github.com/golang/dep/cmd/dep && \
    chown -R 1001 /usr/src/go && \
    chown -R 1001 /usr/local/go && \
    chown -R 1001 ${HOME}/.cache/go-build && \
    rm -f /usr/go${GO_VERSION}.linux-amd64.tar.gz
```